### ISSUE TITLE
`npm install` is not enough to use `npm run test`

### ISSUE THREAD

**Issue by ailyntang** [exercism/javascript#656]
Hi, thanks very much for adding this new exercise. In all the other JS exercises, I have been able to run the tests after typing the following commands into terminal:

1. `npm install`
2. `npm run test`

However with this exercise, I have not been able to run the tests. This is what I have tried:

**First error: `jest: command not found`**
1. `npm install`
2. `npm run test`
Error received: `sh: jest: command not found`

To fix this, I typed in `npm install --save-dev jest`
This fixed the above error. However now a new error appears and I don't know how to fix this.

**New error: `Cannot find module '@babel/preset-env' `**
I can see that there is a file called `babel.config.js` in the folder. Inside the file, I can see the preset configuration.

However I don't know how to fix the error. 
I tried typing: `npm install --save-dev @babel/core @babel/cli` but that did not fix the error.

I have not needed to do anything like this to run tests for the other exercises, so I thought I would raise the issue here. 

Apologies if this is really simple, but I'm learning and tried to Google, but still couldn't find an answer I understood.

Below is the debug log. 
Thanks for your help.
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'test' ]
2 info using npm@6.4.1
3 info using node@v10.15.0
4 verbose run-script [ 'pretest', 'test', 'posttest' ]
5 info lifecycle exercism-javascript@0.0.0~pretest: exercism-javascript@0.0.0
6 info lifecycle exercism-javascript@0.0.0~test: exercism-javascript@0.0.0
7 verbose lifecycle exercism-javascript@0.0.0~test: unsafe-perm in lifecycle true
8 verbose lifecycle exercism-javascript@0.0.0~test: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/ai-lyntang/Exercism/javascript/resistor-color/node_modules/.bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
9 verbose lifecycle exercism-javascript@0.0.0~test: CWD: /Users/ai-lyntang/Exercism/javascript/resistor-color
10 silly lifecycle exercism-javascript@0.0.0~test: Args: [ '-c', 'jest --no-cache ./*' ]
11 silly lifecycle exercism-javascript@0.0.0~test: Returned: code: 1  signal: null
12 info lifecycle exercism-javascript@0.0.0~test: Failed to exec test script
13 verbose stack Error: exercism-javascript@0.0.0 test: `jest --no-cache ./*`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:182:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:182:13)
13 verbose stack     at maybeClose (internal/child_process.js:962:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
14 verbose pkgid exercism-javascript@0.0.0
15 verbose cwd /Users/ai-lyntang/Exercism/javascript/resistor-color
16 verbose Darwin 16.7.0
17 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "test"
18 verbose node v10.15.0
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 1
22 error exercism-javascript@0.0.0 test: `jest --no-cache ./*`
22 error Exit status 1
23 error Failed at the exercism-javascript@0.0.0 test script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by SleeplessByte** [comment]
From your message I can't figure out _which_ exercise; from the log i think it's `resistor-color`. 

If we look at its [package.json](https://github.com/exercism/javascript/blob/master/exercises/resistor-color/package.json), we can see `jest` is already present.

I've also gone ahead an download the exercise again and run the appropriate commands
```
cd resistor-color
npm i
npm test
```

No errors 😓 .

**Comment by ailyntang** [comment]
Thanks so much for looking into this @SleeplessByte . My apologies for not listing the exercise - I thought I had navigated to the correct exercise on GitHub before posting, but I must not have!

You were right, it was indeed `resistor-color`.

And as is always the case, now that you've looked at it, I can no longer replicate the problem either. I think perhaps that `npm install` didn't work properly the first time I did it.

Thanks again, and very sorry to waste your time. I'll close this issue now.
